src/api/gameTitles/gameTitles.router.js

```diff
@@ -3,6 +3,7 @@ import * as gameTitlesController from './gameTitles.controller.js';
 
 const router = Router();
 
+router.get('/all', gameTitlesController.getAllTitles); // this route must be before '/:id'
```

Comment:
The comment shouldn't be submitted to main.

PS: why does it have to be before` /:id` ? 👀

Reply:
'/all' must be before ':id' because if it's the other way, the string "all" in the url WILL be interpreted as the ID by express. 

Another solution would be to change the route to '/' (without 'all').